### ISSUE TITLE
Consider using prettier

### ISSUE THREAD

**Issue by paustint** [Raathigesh/fabulous#18]
**Is your feature request related to a problem? Please describe.**
Because there is a lot of interest in this library and there will possibly be multiple people working on the library, you might consider using prettier for the codebase and adding a `.prettierrc` with your desired rules so that no matter who writes the code, it always gets formatted exactly the same.

this will help keep continuity in the style of the codebase over time.  If you do decide to use prettier, then you may want to put that as a requirement for any PRs to be submitted.

**Comment by Raathigesh** [comment]
Absolutely. Will try to get it in.

**Comment by paustint** [comment]
@Raathigesh - below is a `.prettierrc` configuration file that I have used on other projects and I think it works well.

This is the VSCode prettier extension I prefer [prettier-vscode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
And then we should add `"editor.formatOnSave": true` to the workspace VSCode settings.

In addition, I like to use the [code-spell-checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) extension and I would recommend looking into this.

---
The main thing here is that I have increased the width to 140 characters, because the default is something really small - too small IMO.

```json
{
  "printWidth": 140,
  "semi": true,
  "tabWidth": 2,
  "useTabs": false,
  "singleQuote": true,
  "trailingComma": "es5",
  "bracketSpacing": true,
  "arrowParens": "avoid",
  "insertPragma": false
}
```
